.github/workflows/regression-report.yml

```diff
@@ -33,6 +34,7 @@ on:
 permissions:
   contents: read
   pull-requests: write
+  actions: write
```

Comment:
Please explain

Reply:
not needed, this was my earlier approach of using API. removed